packages/esm-patient-chart-app/src/visit/visit-form/visit-form.scss

```diff
@@ -44,10 +44,15 @@
 .form {
   display: flex;
   flex-direction: column;
-  justify-content: space-between;
+  justify-content: start;
   height: var(--desktop-workspace-window-height);
```

Comment:
This should probably be propagated to more places. We use this logic in most workspace forms.

Reply:
Definitely. I think it's more visible here because we have a slightly larger number of direct children. The other trick here was the `justify-self: end` and `margin-top: auto` applied to the buttons... Otherwise the rendered somewhere in the middle of the form.